package/cfg/rke-cis-1.23-hardened/etcd.yaml

```diff
@@ -177,7 +177,6 @@ groups:
           bin_op: or
           test_items:
             - flag: "--peer-client-cert-auth"
-              set: true
```

Comment:
Are we meant to remove this?

Reply:
Yes this was intentional, yamllint raised an error because of the duplication of `set: true`. And [default cis-1.23/etcd.yaml check:2.5](https://github.com/rancher/security-scan/blob/d781893565af89ba81e85f81d9f5a7324d8f8a01/package/cfg/cis-1.23/etcd.yaml#L85) has only one set: true.
This doesn't block the execution of kube-bench but will fail the validation.

```
$ yamllint rke-cis-1.23-hardened/etcd.yaml | grep -v too | grep duplicate
  185:15    error    duplication of key "set" in mapping  (key-duplicates)
```
For the record (not obviously in this case) test_items are `set: true` by default in [kube-bench] (https://github.com/aquasecurity/kube-bench/blob/fac90f756e5cd976aabcd4ebdb42aad7b58cf6d1/docs/controls.md?plain=1#L225).